vsintegration/src/FSharp.ProjectSystem.FSharp/MenusAndCommands.vsct

```diff
@@ -133,6 +133,15 @@
           <ButtonText>F# Interactive</ButtonText>
         </Strings>
       </Button>
+      
+      <Button guid="FSharpProjectCmdSet" id="cmdidFsDump" priority="0x8000" type="Button">
```

Comment:
This addition should have (but obviously didn't) update all of the `MenusAndCommands.vsct.*.xlf` files.  Can you try running this to force regeneration?

``` bash
msbuild vsintegration\src\FSharp.ProjectSystem.FSharp\ProjectSystem.fsproj /t:UpxateXlf
```